curvefs/src/tools/create/curvefs_create_fs.cpp

```diff
@@ -137,8 +144,9 @@ bool CreateFsTool::AfterSendRequestToHost(const std::string& host) {
                      << ", error text: " << controller_->ErrorText()
                      << std::endl;
         ret = false;
-    } else if (response_->statuscode() == mds::FSStatusCode::OK) {
-        // create success
+    } else if (response_->statuscode() == mds::FSStatusCode::OK ||
+               response_->statuscode() == mds::FSStatusCode::FS_EXIST) {
```

Comment:
maybe fs is truely exist, still return success?

Reply:
> maybe fs is truely exist, still return success?

This is a bug that was fixed by other PRs, and I forgot to modify it here.